### DISCUSSION TITLE
Resolving "unknown key <vscode>" message in checkhealth

### DISCUSSION THREAD

**Discussion by ptdecker** [folke/flash.nvim#299] [Q&A]
Hello!  I have installing LazyVim and after which I ran the suggested "checkhealth".  In the results, I receive the following warning:

```text
lazy: require("lazy.health").check()

lazy.nvim ~
- OK Git installed
- OK no existing packages found by other package managers
- OK packer_compiled.lua not found
- WARNING {flash.nvim}: unknown key <vscode>
```

How can I resolve this warning and what does it mean?

I am using macOS and nvim version 0.9.4.  I am not using vscode but do have it installed on my system.

Thank you for your assistance in advance

**Comment by RiQuY** [comment]
Same issue here, fresh installation of neovim with lazy.vim and this occurs.

**Comment by Spacejet13** [comment]
This happens because in your configuration of `flash.nvim`, you have a property called `vscode` that it does not expect. Try updating your packages with `:Lazy sync` to see if that fixes the issue. If you have not changed this value yourself, it might just be that the distro expects a newer version of `flash.nvim` that supports the tag `vscode` but you have one that does not.

**Reply by ptdecker** [reply]
I'll give this a try

**Reply by Spacejet13** [reply]
Do let me know if this works for you!
